vpr/src/timing/timing_graph_builder.cpp

```diff
@@ -148,11 +383,13 @@ void TimingGraphBuilder::add_block_to_timing_graph(const AtomBlockId blk) {
         }
```

Comment:
I think it is worth adding a comment on line "tnode = tg_->add_node(NodeType::SINK);" that this is a sink because it a capture point (e.g. T setup / hold to clock).

Reply:
I don't believe the comment on line 377 is correct ("//There is an internal combinational connection starting at this sequential input")  makes sense to me.  If there is a combination connection from the sequence input to a part of this block, should simply be an edge from SINK node to the combination destinations?

I believe you need to add to your top-level comment computational + sequence examples.